<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileServer</class>
 <widget class="QMainWindow" name="FileServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小艾服务器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>小艾服务器</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>传输文件</string>
       </attribute>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>641</width>
          <height>151</height>
         </rect>
        </property>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>631</width>
           <height>51</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>100</width>
            <height>45</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>文件路径：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="fileEdit">
          <property name="geometry">
           <rect>
            <x>119</x>
            <y>3</y>
            <width>301</width>
            <height>35</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
         <widget class="QLabel" name="fileSizeLabel">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>0</y>
            <width>161</width>
            <height>45</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>大小：</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>601</width>
           <height>61</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>100</width>
            <height>45</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>发送进度：</string>
          </property>
         </widget>
         <widget class="QProgressBar" name="sendProgressBar">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>10</y>
            <width>350</width>
            <height>45</height>
           </rect>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
         <widget class="QPushButton" name="rebuildPushButton">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>10</y>
            <width>100</width>
            <height>45</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>100</width>
          <height>45</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>信息框:</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>641</width>
          <height>221</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="selectFileBtn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>500</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="startServer">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>500</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>启动服务</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearMsg">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>500</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>清空信息</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>170</width>
          <height>70</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>监听端口：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="portEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>211</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>8888</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
